src/Miunie.Core/Services/PrivacyService.cs

```diff
@@ -0,0 +1,51 @@
+﻿// This file is part of Miunie.
+//
+//  Miunie is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  Miunie is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with Miunie. If not, see <https://www.gnu.org/licenses/>.
+
+using Miunie.Core.Attributes;
+using Miunie.Core.Discord;
+using Miunie.Core.Entities.Discord;
+using Miunie.Core.Json;
+using Miunie.Core.Providers;
+using System.Threading.Tasks;
+
+namespace Miunie.Core
+{
+    [Service]
+    public class PrivacyService
+    {
+        private readonly IDiscordMessages _messages;
+        private readonly IMiunieUserProvider _users;
+        private readonly IJsonParser _jsonParser;
+
+        public PrivacyService(IDiscordMessages messages, IMiunieUserProvider users, IJsonParser jsonParser)
+        {
+            _messages = messages;
+            _users = users;
+            _jsonParser = jsonParser;
+        }
+
+        public async Task OutputUserJsonDataAsync(MiunieUser user)
+        {
+            var userJson = _jsonParser.ConvertToJson(user);
+            await _messages.SendDirectFileMessageAsync(user, userJson, Entities.PhraseKey.USER_PRIVACY_FILE_MESSAGE, user.Name);
```

Comment:
That's a lot of redundant data. :thinking: 
The first two parameters are really the only two needed.
The PhraseKey shouldn't be provided, the Discord layer will infer it from the method being called.
And the username is part of the `user` object, so there is no need for it. :blush: